examples/LexActivator.rb

```diff
@@ -31,9 +31,59 @@ def self.decode_utf16(input)
   module PermissionFlags
     LA_USER = 1
     LA_SYSTEM = 2
+    LA_ALL_USERS = 3
     LA_IN_MEMORY = 4
   end
 
+  BUFFER_SIZE_256 = 256
+  MAX_METADATA_SIZE = 100 
+
+  class OrganizationAddress < FFI::Struct
+    layout :addressLine1, [:char, BUFFER_SIZE_256],
+           :addressLine2, [:char, BUFFER_SIZE_256],
+           :city, [:char, BUFFER_SIZE_256],
+           :state, [:char, BUFFER_SIZE_256],
+           :country, [:char, BUFFER_SIZE_256],
+           :postalCode, [:char, BUFFER_SIZE_256]
+
+    def to_string
```

Comment:
@muneebkq `to_string` is not a very idiomatic name in Ruby, and that `to_hash` would be a more appropriate and descriptive name for a method that converts an FFI::Struct to a Hash.